docs/developing/troubleshooting.md

```diff
@@ -31,6 +31,13 @@ the following HTML to your document's `<head>`:
 <base target="_blank">
 ```
 
+However, if you have both internal and external links, you can add a script
+such as the following which will only set the target on external links:
+
+```js
+window.onload = function(){ var a = document.getElementsByTagName('a'); for (var i=0; i<a.length; i++){ if(a[i].hostname != location.hostname) {a[i].setAttribute('target', '_blank'); } } }
```

Comment:
Let's add some line breaks & indentation, rather than cramming all this in on one line.

Reply:
My original thought was to keep things small, and make this more of a magic "just add this line" thing, since it's part of a larger page, but I have expanded it out and force-pushed the branch.